persistence/javadsl/src/main/java/com/lightbend/lagom/javadsl/persistence/ChildPersistentEntity.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.persistence;
+
+import akka.actor.ActorContext;
+import akka.actor.ActorRef;
+import akka.pattern.Patterns;
+import akka.util.Timeout;
+import com.lightbend.lagom.internal.javadsl.persistence.PersistentEntityActor;
+import com.typesafe.config.Config;
+import play.inject.Injector;
+import scala.compat.java8.FutureConverters;
+import scala.compat.java8.OptionConverters;
+import scala.concurrent.Future;
+import scala.concurrent.duration.Duration;
+import scala.runtime.AbstractFunction0;
+
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * A persistent entity that runs as a child actor of the current actor context.
+ */
+public class ChildPersistentEntity<Command> {
+
+  private final ActorRef actor;
+
+  ChildPersistentEntity(ActorRef actor) {
+    this.actor = actor;
+  }
+
+  /**
+   * Get the actual actor for the entity.
+   * <p>
+   * This may be useful in order to watch the actors termination.
+   */
+  public ActorRef getActor() {
+    return actor;
+  }
+
+  /**
+   * Send a message to the actor.
+   */
+  public void tell(Command msg, ActorRef sender) {
+    actor.tell(msg, sender);
+  }
+
+  /**
+   * Ask the actor with a command.
+   */
+  @SuppressWarnings("unchecked")
+  public <Reply, Cmd extends PersistentEntity.ReplyType<Reply>> CompletionStage<Reply> ask(Cmd msg, Timeout timeout) {
+    return FutureConverters.toJava((Future<Reply>) Patterns.ask(actor, msg, timeout));
+  }
+
+  /**
+   * Forward a message to the actor.
+   */
+  public void forward(Command msg, ActorContext ctx) {
+    actor.forward(msg, ctx);
```

Comment:
I'm not sure I see the purpose of supporting `tell` and `forward` in this feature. `ChildPersistentEntity<Command>` seems to only be used to contain a `PersisentEntity` which only supports `ask`. What's the use case you had in mind?